Sources/CariocaIndicatorConfiguration.swift

```diff
@@ -27,12 +27,12 @@ public protocol CariocaIndicatorConfiguration {
 	///The custom shape of the view
 	func shape(for edge: UIRectEdge, frame: CGRect) -> UIBezierPath
 	///The margins for the icon, depending on the edge
-	func iconMargins(for edge: UIRectEdge) -> (top: CGFloat, right: CGFloat, bottom: CGFloat, left: CGFloat)
+	func iconMargins(for edge: UIRectEdge) -> UIEdgeInsets
 }
 public extension CariocaIndicatorConfiguration {
 	///Default margins are 0,0,0,0
```

Comment:
The default values were 0,0,0,0. Is it still the same ?

Reply:
[Yeah](https://developer.apple.com/documentation/uikit/uiedgeinsets/1625050-init)